Index.html

```diff
@@ -1,231 +1,328 @@
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+				<div id="project4">
+					<a href="#">
+						<img
+							src="./assets/Img-Placeholder.svg"
+							alt="Screenshot of Project"
+						/>
+					</a>
+					<h3 class="multiPost">Multi-Post Stories</h3>
+					<p class="projectBrief">
+						A daily selection of privately personalized reads; no accounts or
+						sign-ups required. has been the industry's standard dummy text ever
+						since the 1500s, when an unknown printer took a standard dummy text.
+					</p>
+					<div class="programmingLanguages">
+						<ul class="languagesUsed">
+							<li>css</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>html</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>bootstrap</li>
+							<li><img src="./assets/separator.svg" alt="Vertical line" /></li>
+							<li>Ruby</li>
+						</ul>
+					</div>
+					<button class="projectButton" type="Button">See Project</button>
+				</div>
+			</div>
+			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
+		</section>
+		<footer class="contact">
+			<div class="contact-text">
+				<h2>CONTACT US</h2>
+				<p>
+					If you have an appication you are interested in developing, a feature
+					that you need built or a pooject that needs coding, i'd love to help
+					with it
+				</p>
+			</div>
+			<form action="https://formspree.io/f/meqnjrwb" method="post">
+				<input
+					name="name"
+					id="name"
+					class="formEl"
+					type="text"
+					placeholder="Your name"
+					maxlength="30"
+					required
+				/>
+				<input
+					name="email"
+					id="email"
+					class="formEl"
+					type="email"
+					placeholder="Your email address"
+					required
+				/>
+				<textarea
+					name="message"
+					id="message"
+					class="formEl"
+					cols="30"
+					rows="10"
+					maxlength="500"
+					placeholder="Please leave your message here"
+					required
+				></textarea>
+				<input class="projectButton btn" type="submit" value="See Project" />
+			</form>
+			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
```

Comment:
In reference to the figma design, the divider has 100% width and spans from left to right. This does not reflect in your design and it would really make the portfolio design better if it matched with that on the figma side.

| Your design | Figma design |
| --- | --- |
| <img width="283" alt="123" src="https://user-images.githubusercontent.com/4247740/144084249-a318d341-b4e9-4a13-926a-787e07dc2369.PNG"> | <img width="282" alt="1232" src="https://user-images.githubusercontent.com/4247740/144084315-df912b53-0b33-4a8b-b89b-688f45f3745a.PNG"> |

Reply:
Well noted. I will work on fixing that. Thanks.